pkg/services/rpcsrv/server.go

```diff
@@ -251,7 +251,10 @@ var invalidBlockHeightError = func(index int, height int) *neorpc.Error {
 
 // upgrader is a no-op websocket.Upgrader that reuses HTTP server buffers and
 // doesn't set any Error function.
-var upgrader = websocket.Upgrader{}
+var upgrader = websocket.Upgrader {
+    // allow & bypass any origin header
+    CheckOrigin: func(r *http.Request) bool { return true },
```

Comment:
> can be configured as feature-flag in configs

Yes, we definitely need a config option for this, we can't just allow any origin (with `false` as default for this config option). Also, I'd prefer to add a list of accepted origins to config as far.

Reply:
Thank for response!
Yeap u right. But right now neo-go node will process request from any origin. The issue is, that it won't process it **if** i pass Origin (because of upgrader behav.), and that's what browser client do 😄 . So merging this PR don't change the logic at all -> just allow browser clients to access nodes.
So it's more a question of adding a **feature** of "u can pass array of origins that will be allowed, other will be restricted"  
Basically just two cases need to be handled: allow all - default behavior | restrict to ones passed in array

Reply:
I'm not sure i spoke clearly. So how neo-go node works right now

Request from client X ---> X don't tell about it's origin. So it's unknown ----> Neo go process it with no issues ✅ 
Request from client B ---> B tells about it's origin, it's `someorigin` ----> Now as origin header passed, Upgrader starts checking it & it's not configured to allow any origin, so it fails the request ❌

The thing is, that it should be so -> because there's no different between client X and B -> neo-go node allow `any` origin right now

So this PR attempt to fix this case 🤔 

But to further improve, we can allow pass in config array of origins -> pass it to upgrader. Then, only requests from those origins will succeed, `all others will be declined`

Reply:
Thank you for the detailed explanation, now I see the problem and seems that this PR solves it. I'll check it out one more time.

Reply:
>  we definitely need a config option for this

It's almost the same problem that `EnableCORSWorkaround` is made for, so it can use the same option.